JsonRpc2/Validator/ValidateVar.php

```diff
@@ -73,27 +73,23 @@ private function bringValueToType($parent, $type, $value)
                 );
```

Comment:
Please review carefully http://www.jsonrpc.org/specification, array-type does not allow.

Reply:
and why you'he commited this in not your topic?
now i cannot merge all changes.